### PR TITLE
Add Goat Attack

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/goatattack/goatattack/issues/47
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/goatattack/goatattack/commit/abf3daa97f96fb1df8f68eec9730dd37e9ea504a https://github.com/goatattack/goatattack/pull/46

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.goatattack.goatattack

File: net.goatattack.goatattack.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+app-id: net.goatattack.goatattack
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
```
Comment (hfiguiere) [inline net.goatattack.goatattack.yaml:3]:
```suggestion
runtime-version: '21.08'
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: net.goatattack.goatattack.yaml (hunk 1 above)
Comment (Mailaender) [reply net.goatattack.goatattack.yaml:3]: That won't work. It is not C++ 17 ready.

Comment (hfiguiere) [reply net.goatattack.goatattack.yaml:3]: well, `cxxflags=-std=c++14`

Comment (Mailaender) [reply net.goatattack.goatattack.yaml]: Yay, that works. Thanks!

**Comment by Mailaender** [comment]
bot, build net.goatattack.goatattack

**Pull request closed without merging**